news/4892.build.rst

```diff
@@ -0,0 +1 @@
+OSX: Fixed the ineffectiveness of the ``--dispath`` argument for the ``BUNDLE`` step.
```

Comment:
Minor typo here; should be `--distpath`, not `--dispath`. Same goes for the commit message.